sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -97,14 +129,17 @@ class EcbTest : public AesTestInitialized {
 
 TEST_F(EcbTest, start) {
   EXPECT_READ32(AES_STATUS_REG_OFFSET, 1);
-  SetExpectedConfig(AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
-                    AES_CTRL_SHADOWED_MODE_VALUE_AES_ECB,
-                    AES_CTRL_SHADOWED_OPERATION_VALUE_AES_ENC,
-                    /*manual_op=*/false,
-                    /*force_zero_masks=*/false);
+  SetExpectedConfig({.key_len = AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
```

Comment:
It might make sense to use a TEST_P for handling the cartesian product of key size and block mode.

See https://github.com/lowRISC/opentitan/blob/master/sw/device/lib/base/math_builtins_unittest.cc#L87